### ISSUE TITLE
IPad issues

### ISSUE THREAD

**Issue by ldellaversano** [BogdanMihaiciuc/BMCollectionView#20]
Hi, in our project, we have problems with IPad use mode.

We are unable to click on dropdwn/folding panel/value diplay widgets when these objects are into BMCollection. We are able to click only button widgets into BMCollection.

Our BMCollection version is 2.5.4 
![image](https://user-images.githubusercontent.com/72251747/94907489-8f46ea80-04a0-11eb-997f-a06cbd30b6e4.png)
Our TWX version is 8.4.3-b2219 .

Do you have any suggestions?

**Comment by BogdanMihaiciuc** [comment]
You should be able to add the `BMCollectionViewCellEventHandler` class to elements to stop collection view cells from stealing their events.

**Comment by ldellaversano** [comment]
Sorry Bogdan, I don't know what BMCollectionViewCellEventHandler is.

Do I find it in gitHub? Do I find it like TWX extension?

Could you explain me how I can add this class?

Thanks a lot.

**Comment by BogdanMihaiciuc** [comment]
It is a CSS class that you can specify in the `CustomClass` property of the widget that you want to click.

**Comment by ldellaversano** [comment]
Hi Bogdan,
It doesn't work.

We set CustomClass of Folding panel and dropdown to BMCollectionViewCellEventHandler but by ipad they(Folding panel and dropdown) don't feel click/tap. 
![image](https://user-images.githubusercontent.com/72251747/94918446-e0f87080-04b2-11eb-8b90-9a1fca27573e.png)

Do you have any other suggestions?

**Comment by BogdanMihaiciuc** [comment]
Does it work if you place the folding panel outside of a collection view?

**Comment by ldellaversano** [comment]
Yes, outside collection folding panel/dropdown works, with or without customClass "BMCollectionViewCellEventHandler ".

**Comment by BogdanMihaiciuc** [comment]
I will need to run some tests in order to look into this. I am assuming you are using the standard **Folding Panel** widget that is available out of the box, right?

**Comment by ldellaversano** [comment]
Yes Bogdan. There is the same behaviour with folding panel, dropdown list, value diplay, all widgets oob.

**Comment by ldellaversano** [comment]
Any news Bogdan? Thanks

**Comment by BogdanMihaiciuc** [comment]
Try updating to collection view + core ui version 2.6. That `BMCollectionViewCellEventHandler` requires that version in order to work.

You will find both of those versions as the latest stable releases for the two widgets.

**Comment by ldellaversano** [comment]
Considering not to update TWX version, staying on 8.4.3, do you think that it is possibile to have some mashup issues after widget upgrade?

**Comment by BogdanMihaiciuc** [comment]
There is no need to update TWX, only the two widgets. Core UI and collection view 2.6 should cleanly upgrade from the previous version with no issues and they should continue to be compatible with previous versions of thingworx; in case there are any issues you can just roll back to 2.5.4.

Note that if you have any other typescript based widgets from the ptc-iot-sharing github (such as SvgViewer, MXGraphDiagram, ThreeViewer, MonacoEditor or others) you should also update those to their latest versions as well.

**Comment by ldellaversano** [comment]
Hi Bogdan, unfortunately the test failed. 

![image](https://user-images.githubusercontent.com/72251747/95838020-9512ba80-0d41-11eb-810c-73a992ea9dbe.png)

We are unable to tap on tablet IoS yet. We obtained an error on chrome console like this 

![image](https://user-images.githubusercontent.com/72251747/95836695-0b162200-0d40-11eb-9c29-ea06acec7ad5.png) 

and the result is that now we see all collection without any graph (and we're unable to tap on dropdown/folding panel). Now after  widgets versions rollback, collections has graphs but we have the main issue yet.

We're waiting your feedback.

Thanks
lda

**Comment by BogdanMihaiciuc** [comment]
One workaround you can use is to use a button to invoke the ```ToggleExpandCollapse``` service of your folding panel as buttons are properly handled by collection view cells.

I will look into adding an exception for the folding panel in a future release.

**Comment by BogdanMihaiciuc** [comment]
I'll release an update to the widget for these cases in the coming days.

**Comment by ldellaversano** [comment]
Hi Bogdan, 
i have some questions about this new release:

1. Will you fix this bug too? 
![image](https://user-images.githubusercontent.com/72251747/96131158-20827c00-0ef9-11eb-8b66-f8dad4dbf8aa.png)
2. Will you start from the latest version or from our version 2.5.4?
3. With new release we will able to tap on dropdown, value display and button but not on folding panel, is it right?

We are waiting your news.

Thanks
lda

**Comment by ldellaversano** [comment]
Hi Bogdan,
Do you have any idea of when you will release new version? 
Will it fix this bug? 
![image](https://user-images.githubusercontent.com/72251747/96859214-8757e600-1461-11eb-99f8-937405a8f19a.png)

Thanks

**Comment by BogdanMihaiciuc** [comment]
I will try to get these changes in around mid next week.
